libs/domain/search/facet-value-navigation/facet-value-navigation.component.spec.ts

```diff
@@ -164,7 +164,7 @@ describe('SearchFacetValueNavigationComponent', () => {
         element = await fixture(
           html`<oryx-search-facet-value-navigation
             enableClear
-            .selectedLength=${mockSelectedLength}
+            dirty
             @oryx.clear=${callback}
           ></oryx-search-facet-value-navigation>`
         );
```

Comment:
description says `and selectedLength is provided`, but there is no `selectedLength` provided. What about testing `selectedLength` and `dirty` props in separate tests?